src/zenml/cli/stack_recipes.py

```diff
@@ -0,0 +1,1013 @@
+    help="Pull stack recipes straight into your current working directory."
+)
+@pass_git_stack_recipes_handler
+@click.argument("stack_recipe_name", required=False, default=None)
+@click.option(
+    "--yes",
+    "-y",
+    "force",
+    is_flag=True,
+    help="Force the redownload of the stack_recipes folder to the ZenML config "
+    "folder.",
+)
+@click.option(
+    "--path",
+    "-p",
+    type=click.STRING,
+    default="zenml_stack_recipes",
+    help="Relative path at which you want to install the stack recipe(s)",
+)
+def pull(
+    git_stack_recipes_handler: GitStackRecipesHandler,
+    stack_recipe_name: str,
+    force: bool,
+    path: str,
+) -> None:
+    """Pull stack_recipes straight into your current working directory.
+
+    Add the flag --yes or -y to redownload all the stack_recipes afresh.
+    Use the flag --version or -v and the version number to specify
+    which version of ZenML you wish to use for the stack_recipes.
+
+    Args:
+        git_stack_recipes_handler: The GitStackRecipesHandler instance.
+        stack_recipe_name: The name of the stack_recipe.
+        force: Force the redownload of the stack_recipes folder to the ZenML config
+            folder.
+        path: The path at which you want to install the stack_recipe(s).
+    """
+    cli_utils.warning(ALPHA_MESSAGE)
+    git_stack_recipes_handler.pull(branch="main", force=force)
+
+    stack_recipes_dir = os.path.join(os.getcwd(), path)
+    io_utils.create_dir_if_not_exists(stack_recipes_dir)
+    try:
+        stack_recipes = git_stack_recipes_handler.get_stack_recipes(
+            stack_recipe_name
+        )
+    except KeyError as e:
+        cli_utils.error(str(e))
+
+    else:
+        for stack_recipe in stack_recipes:
+            destination_dir = os.path.join(os.getcwd(), path, stack_recipe.name)
+            if LocalStackRecipe(
+                name=stack_recipe.name, path=Path(destination_dir)
+            ).is_present():
+                if force or cli_utils.confirmation(
+                    f"Stack recipe {stack_recipe.name} is already pulled. "
+                    "Do you wish to overwrite the directory at "
+                    f"{destination_dir}?"
+                ):
+                    fileio.rmtree(destination_dir)
+                else:
+                    cli_utils.warning(
+                        f"Stack recipe {stack_recipe.name} not overwritten."
+                    )
+                    continue
+
+            cli_utils.declare(f"Pulling stack recipe {stack_recipe.name}...")
+
+            io_utils.create_dir_if_not_exists(destination_dir)
+            git_stack_recipes_handler.copy_stack_recipe(
+                stack_recipe, destination_dir
+            )
+            cli_utils.declare(
+                f"Stack recipe pulled in directory: {destination_dir}"
+            )
+            track_event(
+                AnalyticsEvent.PULL_STACK_RECIPE,
+                {"stack_recipe_name": stack_recipe.name},
```

Comment:
Would it not be nice to also list the stack recipe version here so we can backcheck the version of the recipe?

Reply:
This applies to all track_events in this  PR

Reply:
That's a good point but unfortunately, I haven't implemented versions on the recipes so far. I'll get to it.